package.json

```diff
@@ -21,13 +21,13 @@
     "compile": "NODE_ENV=production gulp compile",
     "test": "gulp validate",
     "gulp": "gulp",
-    "update": "npm update && cd pattern-lab && composer update",
-    "new": "yo ./scripts/new-component/index.js",
-    "setup": "cd pattern-lab && composer install --no-interaction --ignore-platform-reqs",
-    "postinstall": "find node_modules/ -name \"*.info\" -type f -delete",
+    "update": "npm update && cd ./tools/pattern-lab && composer update",
+    "new": "yo ./tools/new-component/index.js",
+    "setup": "cd ./tools/pattern-lab && composer install --no-interaction --ignore-platform-reqs",
     "webpack-pl": "webpack --display-entrypoints --config webpack.pl.config.js",
     "webpack-drupal": "webpack --display-entrypoints --config webpack.drupal.config.js",
-    "webpack-dev-server": "webpack-dev-server --hot --inline --progress"
+    "webpack-dev-server": "webpack-dev-server --hot --inline --progress",
+    "lint:js": "eslint ./source/ ./tools/tasks/ gulpfile.js webpack.*.js --cache"
```

Comment:
This linting script is for manual linting of source files as well. Webpack's rule appears to handle everything else.